test/Microsoft.ML.Core.Tests/UnitTests/TestCSharpApi.cs

```diff
@@ -374,9 +374,9 @@ public void TestCrossValidationMacro()
                         foldGetter(ref fold);
                         Assert.True(ReadOnlyMemoryUtils.EqualsStr("Standard Deviation", fold));
                         if (w == 1)
-                            Assert.Equal(1.584696, stdev, 6);
+                            Assert.Equal(1.584696, stdev, 3);
```

Comment:
Do we need all 6 digits if we are only matching 3 of them? Same for below.

Reply:
I think its better to have the "exact result" as expected and to specify the precision elsewhere. That makes it significantly easier to adjust things, as needed.

Reply:
I agree, it's better to specify Assert.Equal(1.584, stdev) than Assert.Equal(1.584696,3) it just looks weird.

---
In reply to: [227490522](https://github.com/dotnet/machinelearning/pull/1320#discussion_r227490522) [](ancestors = 227490522)

Reply:
okay I will adjust that